File: Examen1/2026_1/Plantillas/PlantillaC#/TestSoft/TestSoftPersistance/Examenes/DAO/ExamenDAO.cs
using TestSoftModel.Examen;
using TestSoftModel.Pregunta;
using TestSoftPersistance.DAO;

namespace TestSoftPersistance.Examenes.DAO
{
    public interface ExamenDAO : IDAO<Examen>
    {
        List<Pregunta> ListarPreguntasPorExamen(int idExamen);
    }
}

File: Examen1/2026_1/Plantillas/TestSoft/TestSoftPersistance/src/main/java/pe/edu/pucp/testsoft/persistance/examen/dao/ExamenDAO.java
package pe.edu.pucp.testsoft.persistance.examen.dao;

import pe.edu.pucp.testsoft.model.examen.Examen;
import pe.edu.pucp.testsoft.model.pregunta.Pregunta;
import pe.edu.pucp.testsoft.persistance.dao.IDAO;

import java.util.List;

public interface ExamenDAO extends IDAO<Examen> {
    List<Pregunta> listarPreguntasPorExamen(int idExamen);
}
